browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1619,7 +1619,10 @@ Are you sure you want to do this?
         Let Brave's developers know that this site doesn't work properly with Shields:
       </message>
       <message name="IDS_REPORT_BROKEN_SITE_TEXT_2" desc="Shields text for broken text warning message.">
-        Note: This site address will be submitted with your Brave version number and your IP address(which will not be stored).
+        Thank you for helping make Brave better for all. This report will only send information necessary for us to fix this site. <ph name="LEARN_MORE">%1$s</ph>
```

Comment:
Maybe "This report will only **contain** information ..."?

Reply:
There is the same string for desktop version: https://github.com/brave/brave-core/blob/master/components/resources/brave_components_strings.grd#L454
In that case we should change both?

Reply:
@mkarolin yeah, I'd agree with that wording change. And yeah, @vadimstruts I think it makes sense to change both Android and Desktop, if they're functionally doing the same thing. So:

`Thank you for helping make Brave better for all. This report will only contain information necessary for us to fix this site.`

Reply:
done